tests/analysis.spec.ts

```diff
@@ -152,6 +155,28 @@ describe('Functional test of analysis', () => {
       TEST_TIMEOUT,
     );
 
+    it('analyze folder legacy json results', async () => {
+      const bundle = await analyzeFolders({
+        connection: { baseURL, sessionToken, source },
+        analysisOptions: { severity: AnalysisSeverity.info, prioritized: true, legacy: true },
+        fileOptions: {
+          paths: [sampleProjectPath],
+          symlinksEnabled: false,
+          maxPayload: 1000,
+          defaultFileIgnores: undefined,
+        },
+      });
+
+      expect(bundle).toBeTruthy();
+      if (!bundle) return; // TS trick
```

Comment:
suggestion: shall we just use @ts-ignore or sth along those line instead?

Reply:
not sure i get you. can you write a suggestion code here?